dojo/tools/deepfence_threatmapper/parser.py

```diff
@@ -0,0 +1,178 @@
+                Kubernetes_Cluster_Name = row.get("Kubernetes Cluster Name")
+                Signature = row.get("Signature")
+                description += "**Filename: **" + Filename + "\n"
+                description += "**Name: **" + Name + "\n"
+                description += "**Rule: **" + Rule + "\n"
+                description += "**Node Name: **" + Node_Name + "\n"
+                description += "**Container Name: **" + Container_Name + "\n"
+                description += "**Kubernetes Cluster Name: **" + Kubernetes_Cluster_Name + "\n"
+                description += "**Content: **" + Content + "\n"
+                description += "**Signature: **" + Signature + "\n"
+                findings.append(
+                    Finding(
+                        title=Name,
+                        description=description,
+                        file_path=Filename,
+                        severity=Severity.capitalize(),
+                        static_finding=False,
+                        dynamic_finding=True,
+                        test=test,
+                    )
+                )
+            elif row.get("@timestamp") and row.get("cve_attack_vector"):
+                cve_attack_vector = row.get("cve_attack_vector")
+                cve_caused_by_package = row.get("cve_caused_by_package")
+                cve_container_image = row.get("cve_container_image")
+                scan_id = row.get("scan_id")
+                cve_container_image_id = row.get("cve_container_image_id")
+                cve_cvss_score = row.get("cve_cvss_score")
+                cve_description = row.get("cve_description")
+                cve_fixed_in = row.get("cve_fixed_in")
+                cve_id = row.get("cve_id")
+                cve_link = row.get("cve_link")
+                cve_severity = row.get("cve_severity")
+                cve_overall_score = row.get("cve_overall_score")
+                cve_type = row.get("cve_type")
+                host_name = row.get("host_name")
+                cloud_account_id = row.get("cloud_account_id")
+                masked = row.get("masked")
+                description += "**cve_attack_vector: **" + cve_attack_vector + "\n"
+                description += "**cve_caused_by_package: **" + cve_caused_by_package + "\n"
+                description += "**cve_container_image: **" + cve_container_image + "\n"
+                description += "**cve_container_image_id: **" + cve_container_image_id + "\n"
+                description += "**cve_description: **" + cve_description + "\n"
+                description += "**cve_severity: **" + cve_severity + "\n"
+                description += "**cve_overall_score: **" + cve_overall_score + "\n"
+                description += "**cve_type: **" + cve_type + "\n"
+                description += "**host_name: **" + host_name + "\n"
+                description += "**cloud_account_id: **" + cloud_account_id + "\n"
```

Comment:
The spacing here seems to mess up the Markdown parser when there's no value included after the `**`. I think all of these properties should be reworked like so:
```suggestion
                description += "**cloud_account_id:** " + cloud_account_id + "\n"
```

![Screenshot 2024-03-08 at 12 53 13](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/4b06692d-aa72-4a54-b0fb-aef3708857d9)

![Screenshot 2024-03-08 at 12 56 04](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/ee991972-07d1-4e46-9b90-528994b8ca74)